src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
Should this maybe be defined on https://github.com/pantsbuild/pants/blob/master/src/python/pants/cache/cache_setup.py#L46 instead? Or on some other subsystem that tasks implicitly depend on (similar to how they implicitly depend on CacheSetup)? The downside would be a more annoying (prefixed) option name... `--cache-test-junit-force` or `--invalidator-test-junit-force`or something.

In my experience, users don't really understand the difference between invalidation and caching... and moving forward, those concepts will continue to merge. So `--cache-*` would make a bit more sense to me.